### PR TITLE
updated utox (0.7.0)

### PR DESCRIPTION
**Pull request by aint**

### CODE REVIEW

File: Casks/utox.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 cask 'utox' do
-  version '0.5.0'
-  sha256 '5aa04ac5454c1786388f18bde60a355b704860debca17faa82eb9c1ecf9846ae'
+  version '0.7.0'
+  sha256 'a982ec72789fae301a80bade047235572e6c636f9216ded6347e75130ba64821'
 
-  # zodiaclabs.org is the official download host per the vendor homepage
-  url "https://zodiaclabs.org/storage/c1/uTox-mac-#{version}.dmg"
+  # github.com/GrayHatter/uTox is the official download host per the vendor homepage
+  url "https://github.com/GrayHatter/uTox/releases/download/v#{version}/uTox-#{version}.dmg"
```
Comment (vitorgalvao) [inline Casks/utox.rb]: Where are you getting this from? I still see the previous URL on the homepage.

File: Casks/utox.rb [hunk 2]
```diff
@@ -1,9 +1,9 @@
 cask 'utox' do
-  version '0.5.0'
-  sha256 '5aa04ac5454c1786388f18bde60a355b704860debca17faa82eb9c1ecf9846ae'
+  version '0.7.0'
+  sha256 'a982ec72789fae301a80bade047235572e6c636f9216ded6347e75130ba64821'
 
-  # zodiaclabs.org is the official download host per the vendor homepage
-  url "https://zodiaclabs.org/storage/c1/uTox-mac-#{version}.dmg"
+  # github.com/GrayHatter/uTox is the official download host per the vendor homepage
```
Comment (vitorgalvao) [inline Casks/utox.rb]: Comment is wrong.

**Comment by vitorgalvao** [comment]
Reverted, since it was merged by mistake. Need the answer to the question before proceeding.

**Comment by aint** [comment]
The latest two uTox versions available only on Github. And now devs in IRC confirmed that GitHub is the new official download host for uTox.

**Comment by vitorgalvao** [comment]
You need to show me something that’s online that proves that repo is official. IRC logs aren’t enough, it needs to be something regular users can audit.

**Comment by aint** [comment]
https://wiki.tox.chat/binaries#mac_os_x

**Comment by vitorgalvao** [comment]
Thank you.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 0397253c5dec69cd2ff8d6c0ef99807137dd206a. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request merged**